test/json-checker.jl

```diff
@@ -0,0 +1,41 @@
+# Run JSON checker tests, excluding tests fail01 and fail18
+
+const JSON_DATA_DIR = joinpath(dirname(dirname(@__FILE__)), "data")
+
+for i in [2:17; 19:33]
+    file = "fail$(lpad(i, 2, 0)).json"
+    filepath = joinpath(JSON_DATA_DIR, "jsonchecker", file)
+    try
+        JSON.parsefile(filepath)
+
+        # should not reach here
+        print_with_color(:red, " [Noncompliant] ")
+        println("Expecting $file to fail, but parsed without error")
```

Comment:
Those tests don't work yet though. I would use @test_broken if that were available, but it hasn't landed yet.